main.py

```diff
@@ -1,14 +1,17 @@
 """`main` is the top level module for your Flask application."""
 
 # Import the Flask Framework
-from flask import Flask, g
-from flask import render_template
+from flask import Flask, g, request
```

Comment:
why are these imports on a different line? probably something i did, can you see if you can put them all on the same line? ``` from flask import Flask, g, request, render_template, redirect, url_for ```